packages/ubus-lime-metrics/files/usr/sbin/last_internet

```diff
@@ -8,7 +8,7 @@
 
 result=''
 # if 8.8.8.8 is accessible, print internet path
-if ping -q -c5 -w10 8.8.8.8 &>/dev/null; then
+if wget http://detectportal.firefox.com/success.txt -q -O - &>/dev/null; then
```

Comment:
It is better we check also the content of the file being "success" because otherwise in case fo having a captive portal in the middle the node will still think internet is available

Reply:
Yes :) I will push a new version, probable as a new script "check_internet` or something like that.

Reply:
If there is a captive portal it must still be considered a gateway. Many times these captive portals are a notification that must be seen once (lack of payment for example) or login screens to the service. If we don't consider them as gateways they can end up being more conflictive than we think.

Reply:
But if you have many gateways, and this one is being captive-portaled, it shouldn't be chosen as a gateway so the others could be enabled.
It should definitely be a third state of "gateway unavailable", where we notify the user that though it seems there could be a gateway on the other side, it is not possible to access it right now.